core/src/main/scala/org/ensime/indexer/FileWatchers.scala

```diff
@@ -155,3 +167,116 @@ private class ApachePollingFileWatcher(
     fm.stop()
   }
 }
+
+object JarJava7Watcher {
+  lazy val jarWatcher = new Java7Watcher()
```

Comment:
just create an instance in the containing class above and pass in as a value. This is going to cause all kinds of trouble in the integration tests.